core/src/script/CGXP/plugins/Print.js

```diff
@@ -577,9 +586,12 @@ cgxp.plugins.Print = Ext.extend(gxp.plugins.Tool, {
                     delete encodedLayer.zoomOffset;
                 }
             }
-            Ext.apply(encodedLayer,
-                encodedLayer.baseURL == this.mapserverURL ?
-                this.encodeLayer : this.encodeExternalLayer);
+            if (encodedLayer.type == 'WMS' || encodedLayer.type == 'wms') {
```

Comment:
just a detail but this could be written 
if (encodedLayer.type.toLowerCase() == 'wms') {